pkg/manager/member/tidb_init_manager.go

```diff
@@ -64,18 +64,7 @@ func NewTiDBInitManager(deps *controller.Dependencies) InitManager {
 }
 
 func (m *tidbInitManager) Sync(ti *v1alpha1.TidbInitializer) error {
-	ns := ti.Namespace
-	tcName := ti.Spec.Clusters.Name
-	tc, err := m.deps.TiDBClusterLister.TidbClusters(ns).Get(tcName)
-	if err != nil {
-		return fmt.Errorf("TidbInitManager.Sync: failed to get tidbcluster %s for TidbInitializer %s/%s, error: %s", tcName, ns, ti.Name, err)
-	}
-	if tc.Spec.TiDB == nil {
-		klog.Infof("TidbInitManager.Sync: Spec.TiDB is nil in tidbcluster %s, skip syncing TidbInitializer %s/%s", tcName, ns, ti.Name)
-		return nil
-	}
-
-	err = m.syncTiDBInitConfigMap(ti)
```

Comment:
Why remove this block?

Reply:
Shocked, I have no impression of it...this doesn't look like it should be deleted.Modify it together with the content to be removed above